docs/api-src/audinterface.rst

```diff
@@ -0,0 +1,36 @@
+audinterface
+============
+
+.. automodule:: audinterface
+
+:mod:`audinterface` provides you classes
+to apply user provided functions to signals or files.
+This is very handy
+if you want to design a user interface for your model.
+You can just use :class:`audinterface.Process`
+and provide your :meth:`model.predict` method
+as ``process_func`` argument.
+
+The :mod:`audinterface` classes implement all
+or a selection of the following methods:
+
+* :meth:`process_file`
+* :meth:`process_files`
+* :meth:`process_folder`
+* :meth:`process_index`
+* :meth:`process_signal`
+* :meth:`process_signal_from_index`
+
+You can inherit from the classes
+or just instantiate them
+to get some standard implementations of those methods.
+The following classes are provided:
+
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+    Feature
```

Comment:
Cool, looks much easier now :)

Reply:
Yes, I also found it nice that we can now simply list all the function/classes here.

The only thing that might be slightly confusing is that we have the manual written files under `docs/api-src` and the automatically created files under `docs/api`. But for debugging it is easier to have separate folders and if something does not work you can also just delete `docs/api` and try again.

Reply:
Yes, we should probably document this.

Reply:
Should I add a section to `CONTRIBUTING.rst` or should we target to document it at our central place at http://devops.pp.audeering.com/sphinx/?

Reply:
Is http://devops.pp.audeering.com/sphinx/ available outside audeering?

Reply:
No, but I think we also have two different use cases of documenting this:

* write documentation on how to setup a new Python package including documentation. For this http://devops.pp.audeering.com/sphinx/ together with the `audfoo` package seems the right place to me
* then in `CONTRBUTING.rst` we could document what you have todo when you add a new function or class, e.g. where to add it in the documentation. But so far, we also didn't do it.

Reply:
So for now I would propose that we stay with the current version of `CONTRIBUTING.rst` and during the upcoming weeks first update the general sphinx documentation and the `audfoo` package. If we then still see the need for adjusting `CONTRIBUTING.rst` in individual packages we can do this as well.

Reply:
ok